client/pages/index.vue

```diff
@@ -192,10 +235,15 @@ export default {
           throw new Error('Network response was not ok');
         }
         const data = await response.json();
-        // console.log("fetchSearchFunction - response JSON", data.features);
+
         this.filteredData = data;
         this.isFetching = false;
         this.error = null;
+
+
+        // console.log("data", data);
+        // console.log("data.features", data.features);
+
```

Comment:
maybe we can also remove thes?

Reply:
I removed these lines for clarity and other logs made for check.